reference/informative/informB15_connection4.rst

```diff
@@ -35,7 +35,7 @@
             </connection>
         </model>
 
-        <!-- Merge the contents to create a valid model -->
+        <!-- This is now valid: The contents were merged to create a valid model -->
```

Comment:
"the" rather than "The" after the column? Otherwise, no full stop at the end of the comment?

Reply:
as above